src/components/Homepage.css

```diff
@@ -0,0 +1,21 @@
+.Homepage, .Parameters{
+  text-align: center;
+  padding-top: 200px;
+  background: url("Homepage_background.png") center bottom no-repeat;
+  background-size: cover;
+  min-height: 700px;
+}
+
+.mainQuestion {
+  padding-top: 30px;
+  margin-bottom: 50px;
+}
+
+.button {
```

Comment:
Can be dangerous, give better class name and follow a convention. Suggestion: [BEM](http://getbem.com/)